src/ape/api/accounts.py

```diff
@@ -123,6 +145,17 @@ def transfer(
         data: Union[bytes, str, None] = None,
         **kwargs,
     ) -> ReceiptAPI:
+        """
+        Make a transfer to an account.
+
+        Args:
+            account (``str``): Concats Address and AddressAPI
+            value (``str``): Concats string and integer
+            data (``str``): Concats bytes and string
```

Comment:
```suggestion
            data (``str``): Extra data.
```

Reply:
Extra data, Extra data read all about it